src/manager.cpp

```diff
@@ -179,4 +186,57 @@ sdbusplus::async::task<>
     co_return;
 }
 
+// NOLINTNEXTLINE
+sdbusplus::async::task<void> Manager::startFullSync()
+{
+    _syncBMCDataIface.full_sync_status(FullSyncStatus::FullSyncInProgress);
+
+    auto fullSyncStartTime = std::chrono::utc_clock::now();
```

Comment:
steady_clock is usually better for keeping track of durations, because it is immune to wall clock changes.  I suspect utc_clock isn't.